doc/configuration/TLS-hardening.md

```diff
@@ -16,7 +16,9 @@ The former one is used primarily by MIM dependencies, while the latter is used o
 None of them is strictly better than the other.
 Below you may find a summary of the differences between them.
 
-* `fast_tls` is faster
+* `fast_tls` used to be faster, however with the progress of OTP TLS implementation
+and additional optimisations applied in MongooseIM this is no longer true.
+* `fast_tls` may use slightly less memory per connection and a a bit less processor time than `just_tls`.
```

Comment:
Actually, now with the improvements from Nelson, memory consumption was the same in our load tests.